pkg/descheduler/strategies/node_taint_test.go

```diff
@@ -228,6 +229,7 @@ func TestDeletePodsViolatingNodeTaints(t *testing.T) {
 			policyv1.SchemeGroupVersion.String(),
 			false,
 			tc.maxPodsToEvictPerNode,
+			tc.maxNoOfPodsToEvictPerNamespace,
```

Comment:
Given `maxNoOfPodsToEvictPerNamespace` is not set in any test case, you can just pass `nil`

Reply:
Fix done ~~ Please check :)